scripts/data_transfer/copy_atac_annos.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gsutil -m mv gs://cpg-tob-wgs-main-upload/open_chromatin_annotation/predicted_l1_celltypes_avg_peaks.csv gs://cpg-tob-wgs-main/tob_wgs_rv/open_chromatin_annotation/
```

Comment:
What does `tob_wgs_rv` stand for? Should we omit that?

Reply:
TOB whole-genome sequencing, rare variant (analysis) - yes I am still fairly unsure re: data paths. For example, this is in principle good for any scRNA-seq from blood, which we'll have for TOB, BioHEART and many others.. but this is the TOB repo anyways, so for now the application when I'll use these data is to prioritise rare variants (from WGS) to test for associations